app/templates/_react.html

```diff
@@ -15,7 +15,6 @@
 {% endblock %}
 
 {% block body_end %}
-  <script type="text/javascript" src="{{ component.get_vendor_bundle() }}" defer></script>
```

Comment:
Removing this as vendor bundle can only be loaded once per page, since the header is on every page already it should load fine